config/fixtures/.env

```diff
@@ -0,0 +1,4 @@
+# This is a comment line
```

Comment:
I think the *recommended* place to put test files is `testdata` (see http://dave.cheney.net/2016/05/10/test-fixtures-in-go)